vendor.conf

```diff
@@ -16,7 +16,7 @@ github.com/gregjones/httpcache 787624de3eb7bd915c329cba748687a3b22666a6
 github.com/json-iterator/go f2b4162afba35581b6d4a50d3b8f34e33c144682
 github.com/peterbourgon/diskv v2.0.1
 github.com/sirupsen/logrus v1.0.0
-github.com/containers/image f0cbc16b444d729362c78244c715b45bf4019b71
+github.com/containers/image 3dda079ae672eb11d7b62e6f85a3283c5b560153
```

Comment:
Could we vendor v1.3 instead of using the commit hash? Maybe, one day, CRI-O will go into Debian as well :-)

Reply:
yup yup

Reply:
done and force pushed